adapters/appnexus/appnexus.go

```diff
@@ -82,52 +82,50 @@ type appnexusReqExt struct {
 var maxImpsPerReq = 10
 
 func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
-	memberIds := make(map[string]bool)
+	memberIds := make(map[string]struct{})
 	errs := make([]error, 0, len(request.Imp))
 
-	// AppNexus openrtb2 endpoint expects imp.displaymanagerver to be populated, but some SDKs will put it in imp.ext.prebid instead
-	var defaultDisplayManagerVer string
-	if request.App != nil {
-		source, err1 := jsonparser.GetString(request.App.Ext, openrtb_ext.PrebidExtKey, "source")
-		version, err2 := jsonparser.GetString(request.App.Ext, openrtb_ext.PrebidExtKey, "version")
-		if (err1 == nil) && (err2 == nil) {
-			defaultDisplayManagerVer = fmt.Sprintf("%s-%s", source, version)
-		}
-	}
-	var adPodId *bool
+	// appnexus adapter expects imp.displaymanagerver to be populated in openrtb2 endpoint
+	// but some SDKs will put it in imp.ext.prebid instead
+	defaultDisplayManagerVer := buildDefaultDisplayManageVer(request)
+
+	var (
+		adPodId         *bool
```

Comment:
adPodId represents an Id not a bool. Should we consider renaming it something that specifies boolean. For example - `hasImpAdPodId` or `IsAdPodIdDefined` or something else.

Reply:
renamed as part of https://github.com/prebid/prebid-server/pull/2632/commits/1750cdf3934b374d82917e4326dc94f3dc39514e